CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Submission Data API
 
+### v6.7.23
+* Add modify submission command with additional products
```

Comment:
please adjust to 
* add new submission command for managing additional products
* improve description